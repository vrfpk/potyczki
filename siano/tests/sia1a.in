10 13
240582 307484 342784 578300 633119 688399 822366 861667 968035 982401
10 0
11 0
12 0
19 0
29 0
36 0
40 0
47 0
54 0
64 0
65 0
67 0
75 0
