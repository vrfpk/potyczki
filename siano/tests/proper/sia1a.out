64251370
6425137
6425137
44975959
64251370
44975959
25700548
44975959
44975959
64251370
6425137
12850274
51401096
